pareto/utilities/get_data.py

```diff
@@ -236,9 +237,41 @@ def get_data(fname, set_list, parameter_list, sum_repeated_indexes=False):
     It is worth highlighting that the Set for time periods "model.s_T" is derived by the
     method based on the Parameter: CompletionsDemand which is indexed by T
     """
+    # Using only the names available in the input sheet
+    set_list_common = []
+    parameter_list_common = []
+    df = pd.ExcelFile(fname)
+    sheet_list = df.sheet_names
+    for name in sheet_list:
+        if name in set_list:
+            set_list_common.append(name)
+        elif name in parameter_list:
+            parameter_list_common.append(name)
+        else:
+            if name != "Overview" and name != "Schematic":
+                warnings.warn(
+                    f"{name} is not found in defined sets or parameters but is parsed in the input data",
+                    UserWarning,
+                    stacklevel=2,
+                )
+
+    for sets in set_list:
+        if sets not in set_list_common:
+            warnings.warn(
+                f"{sets} is defined in set_list but not parsed in the input data",
+                UserWarning,
+                stacklevel=2,
+            )
+    for params in parameter_list:
+        if params not in parameter_list_common:
+            warnings.warn(
+                f"{params} is defined in parameter_list but not parsed in the input data",
+                UserWarning,
+                stacklevel=2,
+            )
     # Reading raw data, two data frames are output, one for Sets, and another one for Parameters
```

Comment:
```suggestion
    # Reading raw data, two data frames are output, one for Sets, and another one for Parameters
    # Pass only tab names that exist in the input file (rather than all expected tab names)
```